nixos/tests/livebook-service.nix

```diff
@@ -0,0 +1,37 @@
+import ./make-test-python.nix ({ lib, pkgs, ...} : {
```

Comment:
Tests are nice :+1: :tada:

Reply:
Agreed and now nobody will break my livebook by updating the package blindly.